pdsfile/pds4file/rules/cassini_uvis_solarocc_beckerjarmak2023.py

```
@@ -0,0 +1,425 @@
+##########################################################################################
+
+description_and_icon_by_regex = translator.TranslatorByRegex([
+    (r'volumes/.*/DATA',                re.I, ('Data files grouped by date', 'CUBEDIR')),
+    (r'volumes/.*/DATA/\w+',            re.I, ('Data files grouped by date', 'CUBEDIR')),
+    (r'volumes/.*/HSP\w+\.DAT',         re.I, ('Time series data',           'DATA')),
+    (r'volumes/.*/HDAC\w+\.DAT',        re.I, ('Binary data cube',           'DATA')),
+    (r'volumes/.*/\w+\.DAT',            re.I, ('Spectral data cube',         'CUBE')),
+    (r'volumes/.*\.txt_[0-9].*',        re.I, ('Text file',                  'INFO')),
+    (r'volumes/.*OLD.DIR',              re.I, ('Directory',                  'FOLDER')),
+    (r'metadata/.*versions\.tab',       0,    ('Table to associate data files with DATA_SET_IDs and versions',
+                                                                             'INDEX')),
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)